addons/website/static/src/snippets/s_searchbar/000.js

```diff
@@ -205,6 +198,16 @@ publicWidget.registry.searchBar = publicWidget.Widget.extend({
             $prevMenu.remove();
         }
     },
+    _getFieldsNames: function () {
```

Comment:
(to respect the style of the file)

```suggestion
    _getFieldsNames() {
```